local-cli/templates/HelloWorld/android/app/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,12 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
 
+    <!-- These permissions are added implicitly by android-jsc, see https://github.com/facebook/android-jsc/pull/12 -->
+    <!-- The following lines can be removed after android-jsc is updated in react-native -->
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" tools:node="remove" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" tools:node="remove" />
```

Comment:
`AsyncStorage` Uses SQLite which saves to the app's local storage which doesn't require a permission.

However `CameraRoll` will require `READ_EXTERNAL_STORAGE` (for `getPhotos`) & `WRITE_EXTERNAL_STORAGE` (for `saveToCameraRoll`)